single_cell_condensed_sdrf.sh

```diff
@@ -203,20 +203,30 @@ use_cell_types_In_condensed() {
       if [ -n "$col_num_ct" ]; then 
         awk -v fieldName="$field_to_extract" -F'\t' 'BEGIN { OFS = "\t" } NR == FNR { cell[$1]; type[$1]=$2; next } $3 in cell { print $1, $2, $3, "factor", fieldName, type[$3] }' \
             <( awk -F'\t' -v cellCol=$col_num_cell_id -v ctCol=$col_num_ct 'BEGIN { OFS = "\t" } { print $cellCol, $ctCol }' $CT ) \
-            <( awk -F'\t' 'BEGIN { OFS = "\t" } {print $1, $2, $3}' $COND | sort | uniq) >> $COND\.with_ct
+            <( awk -F'\t' 'BEGIN { OFS = "\t" } {print $1, $2, $3}' $COND | sort | uniq) > $COND\.with_ct
```

Comment:
why were we appending before and now we are not (to $COND\.with_ct I mean)?

Reply:
Previously, each loop appended to that file, then the combined result was appended to the condensed SDRF, then the whole lot was put through annotation. 

I've just shuffled for simplicity so that $COND\.with_ct is put through the annotate script, and the output of that command is then appended to the condensed.